docs/configuration.md

```diff
@@ -277,6 +277,24 @@ Example:
 CREDENTIAL_VAULT_SECRET_BASE_PATH=secret
 ```
 
+### ENABLE_BROWSER_LOG
+
+Enables addition of browser logs to Allure steps
+
+```conf
+ENABLE_BROWSER_LOG=true
+```
+
+### BROWSER_LOG_BLACKLIST
+
+Blacklists types of browser log entries from appearing in Allure steps.
```

Comment:
Can we add all possible types in the blacklist? are `other` and `console-api` the only possible types?

Reply:
I'm not 100% sure what all of the types are, I've ony been able to trigger those two.

If you can find documentation of the available types then please let me know, I've had no luck.

Reply:
I see. Then let's not mention.